raiden/network/proxies/token_network.py

```diff
@@ -239,8 +241,7 @@ def new_netting_channel(
                     block_identifier=given_block_identifier,
                 )
                 balance = self.token.balance_of(
-                    address=to_checksum_address(self.address),
-                    block_identifier=given_block_identifier,
+                    address=Address(self.address), block_identifier=given_block_identifier
```

Comment:
I was a bit surprise by this downcast, but it is correct. Maybe we should rename `balance` to `network_total_deposit`? (If you, please also change the other functions)